templates/react/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/Create{{pascal_case entry_type.name}}.tsx.hbs

```diff
@@ -0,0 +1,88 @@
+import type { AppClient, Record, EntryHash, AgentPubKey, ActionHash, DnaHash } from '@holochain/client';
+import { FC, useState, useContext, useEffect } from 'react';
+
+import type { {{pascal_case entry_type.name}}{{#each entry_type.fields}}{{#if (eq field_type.type "Enum")}}, {{field_type.label}}{{/if}}{{/each}} } from './types';
+import { HolochainContext } from '../../contexts/HolochainContext';
+{{#uniq_lines}}
+  {{#each entry_type.fields}}
+    {{#if widget}}
+{{> (concat field_type.type "/" widget "/edit/imports") }}
+
+    {{/if}}
+  {{/each}}
+{{/uniq_lines}}
+
+const Create{{pascal_case entry_type.name}}: FC<Create{{pascal_case entry_type.name}}Props> = ({ on{{pascal_case entry_type.name}}Created{{#each entry_type.fields}}{{#if (not widget) }}{{#if (eq cardinality "vector")}}, {{camel_case field_name}}{{else}}, {{camel_case field_name}}{{/if}}{{/if}}{{/each}} }) => {
+  const {client} = useContext(HolochainContext);
+{{#each entry_type.fields}}
+  {{#if widget }}
+    {{#if (not (eq cardinality "vector" ) )}}
+	const [{{camel_case field_name}}, set{{pascal_case field_name}}] = useState<{{> (concat field_type.type "/type")}}>({{> (concat field_type.type "/sample")}});
```

Comment:
I haven't checked out what the other frameworks do re: sample value, but this creates a default experience that causes the user to have to delete a bunch of stuff. Wondering if this value could get moved to the `edit/render.hbs` templates in a `placeholder` field?